src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/DynamicFlags.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.ExpressionEvaluator
+{
+    /// <remarks>
+    /// WARNING: Mutable struct.
+    /// </remarks>
+    internal struct DynamicFlags
```

Comment:
+1...I assume the overhead of these objects wouldn't be huge in normal apps (no dynamic info).